src/cljdoc/server/pedestal_util.clj

```diff
@@ -61,3 +61,15 @@
    {:name ::html
     :enter (fn html-render-inner [context]
              (ok-html context (render-fn context)))}))
+
+(defn body
```

Comment:
Can we think of a better name than `body`? Perhaps `ok-html-or-data`?

Reply:
This could be `coerce-body` as per the reference link I posted in the other comments.